rocAL/include/loaders/video/video_loader.h

```diff
@@ -74,8 +74,8 @@ class VideoLoader : public LoaderModule {
     size_t _sequence_length;
     std::thread _load_thread;
     RocalMemType _mem_type;
-    decoded_image_info _decoded_img_info;
-    decoded_image_info _output_decoded_img_info;
+    DecodedDataInfo _decoded_video_info;
```

Comment:
if decoded_data_info instances are required for all loaders, it is better to define in one place in the base class

Reply:
Moved to the base class